### PR TITLE
Expose csi-driver and cloud-controller-manager logs

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

/area logging
/kind enhancement
/priority normal
/platform gcp

**What this PR does / why we need it**:
Add two new dashboards exposed to the end users which will show logs for `CCM` and the `csi-driver-controller`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
@rfranzke @wyb1 

**Release note**:

```improvement user
New dashboards which expose logs for `cloud-controller-manager` and `csi-driver-controller`.
```

### CODE REVIEW

**Review by vlvasilev — APPROVED** [review]
/lgtm

File: charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-monitoring.yaml [hunk 1]
```diff
@@ -55,8 +55,8 @@ data:
```
Comment (rfranzke) [inline charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-monitoring.yaml:55]: Can you rename the file with an appropriate name? It still has "monitoring" in it, but meanwhile it contains also logging dashboards. Maybe "observability" or something?

File: charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-driver-controller-monitoring-config
```
Comment (rfranzke) [inline charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-monitoring.yaml:4]: Is this really monitoring config?

**Review by rfranzke — COMMENTED** [review]
Thanks, looks good, just some nits about the namings

File: charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-monitoring.yaml (hunk 2 above)
Comment (Kristian-ZH) [reply charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-monitoring.yaml:4]: Maybe also could be renamed to "observability"?

**Review by DockToFuture — DISMISSED** [review]
/lgtm

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**